server/room.ts

```diff
@@ -39,214 +46,115 @@ export class Room {
       this.deserialize(roomData);
     }
 
-    setInterval(() => {
-      // console.log(roomId, this.video, this.roster, this.tsMap, this.nameMap);
+    this.tsInterval = setInterval(() => {
```

Comment:
Are these assignments referenced anywhere? Setting them to a variable is fine, just wondering why the change was needed (for cancellation later?)

Reply:
Yes these are preparations to cancel the intervals later (I also think the interval for the next video needs to be stopped when the next video gets queued so we don't have a problem with double calls since the interval continues running